src/admin/style.scss

```diff
@@ -54,7 +50,7 @@ h1 {
  * Utils
  */
 .gray-700 {
-	color: var(--np-color-wp-gray-700);
+	color: var(--newspack-wizard-color-neutral-70);
```

Comment:
Same here. Why using `--newspack-wizard-color-neutral-70` when we can just leverage `wp-colors.$gray-700`?

Reply:
Refactored 95eba4a67b81c8235b126b765080022a755ab48b